AddProductsCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SimpleInv
```

Comment:
let's avoid abbreviations especially when the term is not excessively long. 
we typically reserve abbreviations for terms composed of multiple words and are commonly used within the project domain.